src/asm/Makefile

```diff
@@ -1,12 +1,12 @@
 NAME	= asm
-SRC	= main.c op.c file_op.c stdin.c get_next_line.c whitespace.c util.c
-OBJ	= $(SRC:%.c=%.o)
-CC	= gcc
+SRC		= char_array.c count.c error.c file_op.c instruction_utils.c instructions.c is_operand.c label_utils.c main.c op.c output.c queue_add.c resolve_labels.c string.c trim.c util.c validate_header.c whitespace.c header_utils.c hexdump.c operand_error.c
```

Comment:
In the near future, let's remove this Makefile since we have the one in the root.